u2i-ci_utils.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'rubocop-rspec'
   spec.add_dependency 'rubocop-checkstyle_formatter'
 
-  spec.add_dependency 'simplecov', '~> 0.11.0'
+  spec.add_dependency 'simplecov', '~> 0.14.1'
```

Comment:
Mr "I give talks on RB conferences" this syntax is is locking on minor version :D